src/renderers/shared/stack/event/EventConstants.js

```diff
@@ -87,7 +87,9 @@ var topLevelTypes = keyMirror({
   topVolumeChange: null,
   topWaiting: null,
   topWheel: null,
-});
+};
+
+export type TopLevelTypes = $Enum<typeof topLevelTypes>;
```

Comment:
I found out about this by going through the internal flow support group. Let me ping Flow people about this